src/main/java/com/epam/pages/popup/AcademicYearsAndVacationPopup.java

```diff
@@ -1,6 +1,112 @@
 package com.epam.pages.popup;
 
-import com.epam.pages.common.CommonPopup;
+import com.epam.helpers.ErrorMessagesProvider;
+import com.epam.helpers.SharedTestData;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.Select;
 
-public class AcademicYearsAndVacationPopup extends CommonPopup {
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.*;
+
+public class AcademicYearsAndVacationPopup extends CreatePopup {
```

Comment:
Extend this class from CommonPopup, because you needn't fields or methods from CreatePopup

Reply:
done